testing/kuttl/e2e-other-gke/cluster-migrate/11--delete.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+delete:
+  - apiVersion: postgres-operator.crunchydata.com/v1beta1
+    kind: PostgresCluster
+    name: cluster-migrate
```

Comment:
🤔 We don't usually delete. Is there a benefit to it?

Reply:
This came up as a possible topic around our CI best practice: KUTTL will delete ns (unless set not to), but it doesn't consider deletion of namespace as part of the test, so it doesn't wait for the ns to be gone before starting the next test.

Reply:
That said, this delete was actually doing double-duty:
1) it's _maybe_ best practice for our CI system
2) I was deleting this cluster and making sure it was gone before manually deleting the PV. (I'm not manually deleting any more, since we switch the Reclaim policy back to what it was before.)

Reply:
After further discussion, I'm removing the delete for now: this will either need to be a convention we decide on (either always or never deleting), or possibly a change in KUTTL, which could be changed to wait for the ns to delete before marking the test as done.